actions/docker.go

```diff
@@ -0,0 +1,75 @@
+package actions
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/google/go-github/github"
+
+	reg "github.com/deis/deisrel/registry"
+)
+
+const (
+	ciRepoOrg    = "deisci"
+	prodRepoOrg  = "deis"
+	monitorImage = "monitor"
+)
+
+// TODO: place in 'docker' package, https://github.com/deis/deisrel/issues/60
+func dockerCheckTags(ghClient *github.Client, quay reg.Registry, hub reg.Registry, repoAndShas []repoAndSha, ref string) ([]string, []error) {
+	var wg sync.WaitGroup
+	done := make(chan bool)
+	resultsCh := make(chan string)
+	errCh := make(chan error)
+	defer close(errCh)
```

Comment:
unless I'm reading this wrong, this close may result in a panic if one of the goroutines are still writing to `errCh` when the close happens.